tests/textcode/test_analysis.py

```diff
@@ -139,11 +145,11 @@ def test_numbered_text_lines_return_correct_number_of_lines(self):
         assert 2 == len(result)
 
     def test_as_unicode_converts_bytes_to_unicode(self):
-        test_line =  b'    // as defined in https://tools.ietf.org/html/rfc2821#section-4.1.2”.'
```

Comment:
It is essentially to keep bytes in this test... the whole purpose is to tests conversion from bytes to unicode!